examples/http/ssl_certificate.js

```diff
@@ -0,0 +1,107 @@
+
+/**
+ * This driver checks the status of https certificate for the target device
+ * Communication protocol is https
+ * return the certificate Issuer, Expiry, Remaining days, Is valid, Certificate authorisation error
+ */
+
+var _var = D.device.createVariable;
+var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+
+/**
+ * 
+ * @returns Promise that wait for the https call to the target machine and parse the needed data
+ */
+function getCertificateData() {
+    var d = D.q.defer();
+    D.device.http.get(
+        {
+            url: "/",
+            headers: {
+                connection: "keep-alive",
+            },
+            rejectUnauthorized: false,
+            protocol: "https"
+        }, function (err, resp) {
+            if(err){
+                console.error(err);
+                D.failure();
+            }
+            var cert = resp.connection.getPeerCertificate();
+            d.resolve({
```

Comment:
You should add a conditional similar to the bulk ssl driver otherwise cert.issuer.O can crash the run

Reply:
It also depends on the server if the TCP connection will be re-used and in such cases the peer certificate does not have values

Reply:
in bluk case, I'm ignoring the servers that have a problem without breaking the driver with an error